<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectContactDialog</class>
 <widget class="QDialog" name="SelectContactDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>408</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SelectContactDialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetBorder" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="TitlebarWidget" name="titlebarWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="border" native="true">
        <widget class="QPushButton" name="addBtn">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>100</y>
           <width>75</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>--&gt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="delBtn">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>160</y>
           <width>75</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;--</string>
         </property>
        </widget>
        <widget class="QListWidget" name="rightWidget">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>10</y>
           <width>221</width>
           <height>301</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="btnOK">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>320</y>
           <width>75</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="btnCancel">
         <property name="geometry">
          <rect>
           <x>510</x>
           <y>320</y>
           <width>75</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>261</width>
           <height>25</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
        <widget class="RosterTreeView" name="leftTreeView">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>261</width>
           <height>261</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TitlebarWidget</class>
   <extends>QWidget</extends>
   <header>TitlebarWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RosterTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">rostertreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
